auth-service/src/main/kotlin/uk/gov/communities/delta/auth/security/RateLimiting.kt

```diff
@@ -0,0 +1,35 @@
+package uk.gov.communities.delta.auth.security
+
+import io.ktor.server.application.*
+import io.ktor.server.plugins.*
+import io.ktor.server.plugins.forwardedheaders.*
+import io.ktor.server.plugins.ratelimit.*
+import org.slf4j.LoggerFactory
+import uk.gov.communities.delta.auth.config.Env
+import kotlin.time.Duration.Companion.minutes
+
+fun Application.configureRateLimiting() {
+    val logger = LoggerFactory.getLogger("Application.RateLimiting")
+    val rateLimit = Env.getRequiredOrDevFallback("AUTH_RATE_LIMIT", "10").toInt()
```

Comment:
Pass the rate limit in to the function rather than reading it here, that way we can change it in tests (so we don't have to make 10 requests, and they won't break if we change this default value)

Reply:
Would be good to log the rate limit on startup too